doc/source/drivers/vector/amigocloud.rst

```diff
@@ -121,6 +112,16 @@ The following layer creation options are available:
 -  **GEOMETRY_NULLABLE**\ =YES/NO: Whether the values of the geometry
    column can be NULL. Defaults to YES.
 
+Configuration options
+---------------------
+
+The following :doc:`configuration options <../../user/configoptions>` are 
```

Comment:
Would be best to link it to it with :ref: to avoid hardcoding the path . The same remark applies to a number of other similar places:
```suggestion
The following :ref:`configuration options <configoptions>` are 
```